vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl

```diff
@@ -11134,23 +11164,28 @@ quantitykind:MechanicalEnergy
 .
 quantitykind:MechanicalImpedance
   a qudt:QuantityKind ;
+  qudt:hasDimensionVector qkdv:A0E-2L2I0M1H0T-3D0 ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
   rdfs:label "Mechanical Impedance"@en ;
+  skos:broader quantitykind:Impedance ;
 .
 quantitykind:MechanicalMobility
   a qudt:QuantityKind ;
   qudt:applicableUnit unit:MOHM ;
+  qudt:hasDimensionVector qkdv:A0E0L0I0M-1H0T1D0 ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
   rdfs:label "Mechanical Mobility"@en ;
 .
 quantitykind:MechanicalSurfaceImpedance
   a qudt:QuantityKind ;
+  qudt:hasDimensionVector qkdv:A0E-2L2I0M1H0T-3D0 ;
```

Comment:
I'm finding Acoustic impedance to have dimension Mass/Length**3 . time, so this would yield Mass/Time.
But I could be wrong.

Reply:
There was an applicable unit of MOHM so I took the dimension vector from it, which is qkdv:A0E0L0I0M-1H0T1D0.

Reply:
Take another look.

Reply:
Does this still have a skos:broader of quantitykind:Impedence? If so then the dimension vectors should be the same, right?

Reply:
MechanicalImpedance = AcousticImpedance x Area, so those two seem correct.

...and to answer your question, Impedance is an electrical concept, so no, it is not skos:broader than any of the mechanical analogs. Wikipedia: "There is a [close analogy](https://en.wikipedia.org/wiki/Mechanical%E2%80%93electrical_analogies#Impedance_analogies) with [electrical impedance](https://en.wikipedia.org/wiki/Electrical_impedance), which measures the opposition that a system presents to the [electric current](https://en.wikipedia.org/wiki/Electric_current) resulting from a [voltage](https://en.wikipedia.org/wiki/Voltage) applied to the system."

Which leaves the hard one, MechanicalSurfaceImpedance. [This source](https://apps.dtic.mil/sti/pdfs/ADA315595.pdf) implies it is dimensionally equivalent to pressure/velocity, which by my calculations is Mass/(time x length**2). [Other sources](https://www.wikidata.org/wiki/Q6421317) state it is just force/speed (i.e. not force per unit area/speed), which is mass/time, and is synonymous with MechanicalImpedance. Yet another source, listed [here](https://www.sciencedirect.com/topics/engineering/mechanical-impedance) defines mechanical impedance Z as the square root of (pressure/velocity)!

 Having run out of any other sources, I suggest we define it dimensionally as pressure/speed with DV of Mass/(time x length**2) and point to the [first reference](https://apps.dtic.mil/sti/pdfs/ADA315595.pdf). Perhaps we should include in the comment that we are seeking resolution.

Reply:
Fixed